src/Core/Managed/PCL/project.json

```diff
@@ -0,0 +1,10 @@
+﻿{
+  "supports": {},
+  "dependencies": {
+    "Microsoft.NETCore.Portable.Compatibility": "1.0.1",
```

Comment:
Yup. These seem to be safe to delete. It works fine without it. On the consuming side, the dependencies are automatically picked up.
I'll wait for the results of your verification before making this change.